apps/web/app/app.dub.co/(dashboard)/[slug]/(ee)/program/network/use-partner-network-filters.tsx

```diff
@@ -53,57 +48,11 @@ export function usePartnerNetworkFilters({
               right: nFormatter(_count, { full: true }),
             })) ?? [],
       },
-      {
-        key: "industryInterests",
-        icon: Heart,
-        label: "Industry interest",
-        multiple: true,
-        options:
-          industryInterests?.map(({ id, icon: Icon, label }) => ({
-            value: id,
-            label,
-            icon: <Icon className="size-4" />,
-          })) ?? [],
-      },
-      {
-        key: "salesChannels",
-        icon: InvoiceDollar,
-        label: "Sales channel",
-        multiple: true,
-        options:
-          salesChannels?.map(({ id, label }) => ({
-            value: id,
-            label,
-          })) ?? [],
-      },
-      {
-        key: "preferredEarningStructures",
-        icon: MoneyBills2,
-        label: "Preferred earning structure",
-        multiple: true,
-        options:
-          preferredEarningStructures?.map(({ id, label }) => ({
-            value: id,
-            label,
-          })) ?? [],
-      },
     ],
     [countriesCount],
   );
 
-  const multiFilters = useMemo(
-    () => ({
-      industryInterests:
-        searchParamsObj.industryInterests?.split(",")?.filter(Boolean) ?? [],
-      salesChannels:
-        searchParamsObj.salesChannels?.split(",")?.filter(Boolean) ?? [],
-      preferredEarningStructures:
-        searchParamsObj.preferredEarningStructures
-          ?.split(",")
-          ?.filter(Boolean) ?? [],
-    }),
-    [searchParamsObj],
-  );
+  const multiFilters = useMemo(() => ({}), []);
```

Comment:
I think we need to either remove `multiFilters` completely or add an `as Record<string, string[]>` here to prevent the type error below in `activeFilters`